### PR TITLE
Decode Morse: Message # P1

### PR DESCRIPTION
**Pull request by yigitm**
We added/done below features/actions

- Add rubocop gem
- Add readme
- Add decode method

Thanks you for your time.

### CODE REVIEW

**Review by julie-ify — CHANGES_REQUESTED** [review]
Hi @yigitm, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has commits from only one partner. Please ensure all partners make contributions to this project by having equal commits. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ cd morse-code-decoder
 Run the app with below command.
```
Comment (elisha2kyakpo1) [inline README.md:49]: - [x] [OPTION AL] It can be good if a readme file contain the project title.

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (yigitm) [reply README.md:49]: Thanks. Readme is updated.

**Pull request merged**